tree/30_generic_methods/file_check_block_device.cf

```diff
@@ -0,0 +1,37 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+# Copyright 2016 Janos Mattyasovszky
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name File check is block device
+# @description Checks if a file exists and is a block device
+#
+# @parameter file_name File name
+# 
+# @class_prefix file_check_special_block_device
```

Comment:
Since this bundle doesn't do any result class definition itself, this class_prefix should exactly match the class_prefix of the underlying method, file_check_special.